vhdbuilder/packer/components.json

```diff
@@ -48,7 +48,6 @@
       "downloadURL": "mcr.microsoft.com/containernetworking/cni-dropgz:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
-        "v0.0.4",
         "v0.0.4.1",
         "v0.0.9"
```

Comment:
looks like we can replace v0.0.9 -> v0.0.13

Reply:
not .13, should be .11

Reply:
done